src/components/SituationFinish/index.tsx

```diff
@@ -22,6 +28,49 @@ const Finish: React.FC<FinishProps> = ({ topics }) => {
     navigate(0);
   };
 
+  const goToMain = () => {
+    navigate("/");
```

Comment:
```suggestion
    navigate(PAGE_PATH.MAIN);
```
저희 path 상수화 해둔거 있어서 그거 사용해도 좋을 것 같아요~!